src/test/java/org/ays/tests/institution/assignmentmanagementservice/PostAssignmentTest.java

```diff
@@ -110,7 +110,7 @@ public void createAssignmentWithInvalidLongitude(Double invalidLongitude, String
     }
 
 
-    @Test(dataProvider = "invalidLatitudeValues")
+    @Test(dataProvider = "invalidLatitudeValues",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html